src/main/steps/c100-rebuild/child-details/navigationController.ts

```diff
@@ -7,12 +7,17 @@ import {
   C100_CHILDERN_DETAILS_PARENTIAL_RESPONSIBILITY,
   C100_CHILDERN_DETAILS_PERSONAL_DETAILS,
   C100_CHILDERN_FURTHER_INFORMATION,
+  C100_CHILDERN_OTHER_CHILDREN_NAMES,
+  C100_CHILDERN_OTHER_CHILDREN_PERSONAL_DETAILS,
+  C100_CONFIDENTIALITY_DETAILS_KNOW,
   PageLink,
 } from '../../urls';
 
 class ChildrenDetailsNavigationController {
   private childrenDetails: ChildrenDetails[] | [] = [];
 
+  private otherChildrenDetails: ChildrenDetails[] | [] = [];
+
   private childId: ChildrenDetails['id'] = '';
 
   private getNextChild(): ChildrenDetails | null {
```

Comment:
Move getNextChild to utils and use it commonly for both child-details and other-children

Reply:
I dont think you can use the same function for this @vivek-sekhar

Reply:
I dont think you can use the same function for this